### DISCUSSION TITLE
希望支持 databendwriter 不知道是否可以接受外部 pr 呢？

### DISCUSSION THREAD

**Discussion by TCeason** [wgzhao/Addax#713] [Ideas]
[Databend](https://databend.rs/doc) 是一个开源弹性和工作负载感知的现代云数据仓库。

通过对一些 ETL 工具进行评测后，我们发现 Addax 是一款非常先进，且框架清晰的产品。基于此，我们希望可以提供 databendwriter 。不知道是否可以接受外部 pr ？@wgzhao

**Comment by wgzhao** [comment]
当然接受
非常欢迎
